src/graphnet/data/dataset/sqlite/sqlite_dataset_perturbed.py

```diff
@@ -78,6 +80,10 @@ def __init__(
                 in this case to events with no value in the corresponding
                 table/column. That is, if no per-event loss weight table/column
                 is provided, this value is ignored. Defaults to None.
+            seed: Optional seed for random number generation (int or numpy Generator). 
```

Comment:
It seems the lines in the docstring are too long. Since we're using type hints it is not necessary to include the allowed values in the docstring.